src/WordPress/Blueprints/Runtime/NativePHPRuntime.php

```diff
@@ -32,14 +33,7 @@ public function getDocumentRoot(): string {
 	}
 
 	public function resolvePath( string $path ): string {
-		if ( ! strlen( $path ) ) {
-			return $this->getDocumentRoot();
-		}
-		if ( $path[0] === '/' ) {
-			return $path;
-		}
-
-		return join_paths( $this->getDocumentRoot(), $path );
+        return Path::makeAbsolute($path, $this->getDocumentRoot());
```

Comment:
Nice! Just a nitpick – let's indent this line. Code formatting is still messy in general and this repo needs a formatter, but let's keep at least the indentation in tact.

Reply:
Yeah. I've noticed that after the push. But - YES, PLEASE, a formatter is a must.

Reply:
It's probably due to mixing tabs and spaces